apps/condo/domains/meter/hooks/useImporterFunction.ts

```diff
@@ -285,18 +297,18 @@ export const useImporterFunctions = (): [Columns, RowNormalizer, RowValidator, O
     const meterReadingCreator: ObjectCreator = async ({ row, addons }: ProcessedRow) => {
         if (!row) return Promise.resolve()
         const [
-            address,
+            , // address
             unitName,
-            unitType,
+            , // unitType
             accountNumber,
-            meterType,
+            , // meterType
             meterNumber,
             numberOfTariffs,
-            value1,
-            value2,
-            value3,
-            value4,
-            readingSubmissionDate,
+            , // value1
```

Comment:
Why this lines become commented? Could it be used in the future?

Reply:
This is to avoid getting too many warnings about `Unused constant` from `ESLint`

Reply:
you can use them if you need